docs/stdlib/json.rst

```diff
@@ -245,9 +240,10 @@ possible to cast a JSON value directly into a :eql:type:`tuple`.
 
 .. eql:operator:: jsonobjdest: json [ str ] -> json
 
-    JSON object property access.
+    Accesses the value of a :eql:type:`json` from its :eql:type:`str` key:
```

Comment:
```suggestion
    Accesses an element of a :eql:type:`json` object given its key:
```
- Since the "value" of the `json` would be the entire object, I've swapped that word for "element" to clarify this will access only a component of the overall `json` value.
- Clarified this operator applies only to `json` objects